jerry-core/CMakeLists.txt

```diff
@@ -250,6 +250,16 @@ if (FEATURE_VM_EXEC_STOP)
   set(DEFINES_JERRY ${DEFINES_JERRY} JERRY_VM_EXEC_STOP)
 endif()
 
+# Sleep function availability check
+INCLUDE (CheckIncludeFiles)
+CHECK_INCLUDE_FILES (time.h HAVE_TIME_H)
+CHECK_INCLUDE_FILES (unistd.h HAVE_UNISTD_H)
+if(HAVE_TIME_H)
+  set(DEFINES_JERRY ${DEFINES_JERRY} HAVE_TIME_H)
+elseif(HAVE_UNISTD_H)
+  set(DEFINES_JERRY ${DEFINES_JERRY} HAVE_UNISTD_H)
+endif()
```

Comment:
all this is debugger specific. please move this to line 184 under the debugger feature check